website/src/main/webapp/script.js

```diff
@@ -4,22 +4,27 @@
  * fetches names of three other orgs
  */
 function findSimilarOrgs() {
```

Comment:
Does this function still have the same behavior as before? If not, maybe it would be better to either break it up into helper functions or write a different function rather than editing the function itself. Other functions and web pages may depend on this same function. If the behavior is the same and you are just refactoring then please ignore my comment